frontend/packages/core/src/Assets/global.ts

```diff
@@ -32,3 +32,5 @@ export const StyledSVG = styled.svg<{ size?: IconSizeVariant }>(props => ({
 export interface SVGProps extends React.SVGProps<SVGSVGElement> {
   size?: IconSizeVariant;
 }
+
+export const SEVERITIES = ["error", "info", "success", "warning"];
```

Comment:
Lets either drop this into the note component or hard code it into the note.stories, the Assets/global is more so for SVG Assets, so it doesn't really apply here